dir.props

```diff
@@ -268,6 +270,14 @@
     <IlasmToolPath Condition="'$(OSEnvironment)'=='Windows_NT'">%WINDIR%\Microsoft.NET\Framework\v4.0.30319\ilasm.exe</IlasmToolPath>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
This is needed since we are no longer updating the BuildTools version. It configures the various roslyn toolset properties as appropriate.